cfy_manager/utils/systemd.py

```diff
@@ -59,8 +59,9 @@ def configure(self, service_name, render=True, tmpfiles=False):
 
         logger.debug('Deploying systemd EnvironmentFile...')
         deploy(env_src, env_dst, render=render)
-        logger.debug('Deploying systemd .service file...')
-        deploy(srv_src, srv_dst, render=render)
+        if exists(srv_src):
```

Comment:
I think we should add a comment here to explain when the unit file might exist and when not, and maybe even a TODO to remove it when everything is done in the RPMs